CMakeLists.txt

```diff
@@ -94,6 +94,12 @@ therock_enable_external_source("composable-kernel" "${THEROCK_SOURCE_DIR}/ml-lib
 # Overall build settings.
 option(THEROCK_VERBOSE "Enables verbose CMake statuses" OFF)
 
+# Map THEROCK_VERBOSE to a CLI flag for fileset_tool.py
+set(THEROCK_FILESET_VERBOSE_ARG "")
+if(THEROCK_VERBOSE)
+  set(THEROCK_FILESET_VERBOSE_ARG --verbose)
+endif()
```

Comment:
I think I'd prefer putting this closer to the callsite(s) in `cmake/therock_subproject.cmake`, to keep the top level `CMakeLists.txt` simpler. If there were multiple uses in different files then sharing like this would make more sense.

I do see a case for keeping it near where `THEROCK_VERBOSE` is set though.

Reply:
Updated! Moved it in  `cmake/therock_subproject.cmake`